deployments/mainnet/usdc/migrations/1659582050_raise_supply_caps_and_seed_reserves_and_rewards.ts

```diff
@@ -0,0 +1,129 @@
+import { CometInterface, CometProxyAdmin, CometRewards, Configurator, ERC20, GovernorSimple, IGovernorBravo, ProxyAdmin, SimpleTimelock } from '../../../../build/types';
+import { DeploymentManager } from '../../../../plugins/deployment_manager/DeploymentManager';
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import { debug } from '../../../../plugins/deployment_manager/Utils';
+import { extractCalldata } from '../../../../src/utils';
+import { exp } from '../../../../test/helpers';
+
+
+interface Vars { };
+
+export default migration('1659582050_raise_supply_caps_and_seed_reserves_and_rewards', {
+  prepare: async (deploymentManager: DeploymentManager) => {
+    return {};
+  },
+
+  enact: async (deploymentManager: DeploymentManager, vars: Vars) => {
+    const { ethers } = deploymentManager.hre;
+
+    // XXX this only has the ABI for the governor proxy
+    const govProxy = await deploymentManager.contract('governor');
+    const governor = await ethers.getContractAt(
+      'IGovernorBravo',
+      govProxy.address,
+    ) as IGovernorBravo;
+    const comet = await deploymentManager.contract('comet') as CometInterface;
+    const configurator = await deploymentManager.contract('configurator') as Configurator;
+    const proxyAdmin = await deploymentManager.contract('cometAdmin') as CometProxyAdmin;
+    const rewards = await deploymentManager.contract('rewards') as CometRewards;
+    const USDC = await deploymentManager.contract('USDC') as ERC20;
+    const COMP = await deploymentManager.contract('COMP') as ERC20;
+    const WBTC = await deploymentManager.contract('WBTC') as ERC20;
+    const WETH = await deploymentManager.contract('WETH') as ERC20;
+    const UNI = await deploymentManager.contract('UNI') as ERC20;
+    const LINK = await deploymentManager.contract('LINK') as ERC20;
+
+    // XXX assert supply caps are currently 0
+
+    // Steps:
+    // 1. Increase supply caps for each of the assets
+    // 2. Increase borrow reward speed
+    // 3. Deploy and upgrade to a new version of Comet
+    // XXX these supply caps might be a bit too high
+    const updateCOMPSupplyCapCalldata = extractCalldata((await configurator.populateTransaction.updateAssetSupplyCap(comet.address, COMP.address, exp(50_0000, 18))).data);
```

Comment:
This says 500K not 50K, right? Which is a weird way to right and prob 5x too large? Will fix on my rebase

Reply:
Hm wondering now if these were actually needed to pass our scens